### PR TITLE
[Course Task] cloud-pkg-cloudstream-config

### PR DESCRIPTION
**Pull request by jxs1211**
Signed-off-by: xian-jie.shen <327411586@qq.com>

**What type of PR is this?**
ut

**What this PR does / why we need it**:
[122](https://github.com/kubeedge/community/issues/122)

**Which issue(s) this PR fixes**:
none

**Special notes for your reviewer**:
```
go test config_test.go config.go -cover -coverprofile=cover.out && go tool cover -func=cover.out && go tool cover -html=cover.out -o coverage.html
ok      command-line-arguments  0.008s  coverage: 90.9% of statements
/home/going/workspace/kubeedge/cloud/pkg/cloudstream/config/config.go:40:       InitConfigure   90.9%
total:                                                                          (statements)    90.9%
```

**Does this PR introduce a user-facing change?**:
none

### CODE REVIEW

File: cloud/pkg/cloudstream/config/config_test.go [hunk 1]
```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2020 The KubeEdge Authors.
```
Comment (vincentgoat) [inline cloud/pkg/cloudstream/config/config_test.go:2]: The copyright date should be updated to 2022.

File: cloud/pkg/cloudstream/config/config_test.go [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+import (
+	"encoding/pem"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/cloudcore/v1alpha1"
+)
+
+var pemData = `verify return:0
```
Comment (vincentgoat) [inline cloud/pkg/cloudstream/config/config_test.go:28]: The pemData is too complex, is this necessary?

Comment (jxs1211) [reply cloud/pkg/cloudstream/config/config_test.go:28]: I tried to make it simpler, but it didn't work, the `Decode` of the process is quite complicated to me, and I'd be thankful if you could show a simple example or give some tips.

Comment (jxs1211) [reply cloud/pkg/cloudstream/config/config_test.go]: updated

File: cloud/pkg/cloudstream/config/config_test.go (hunk 1 above)
Comment (jxs1211) [reply cloud/pkg/cloudstream/config/config_test.go:2]: updated

**Comment by jxs1211** [comment]
@fisherxu Could this PR be merged?

**Pull request closed without merging**